From 813a8158546bc213f9da408fb970d9c7ba0ec9c2 Mon Sep 17 00:00:00 2001
From: Weiwen Chen <cww@rock-chips.com>
Date: Fri, 15 May 2020 10:27:24 +0800
Subject: [PATCH] ARM: configs: rv1126_defconfig: enable build wifi module

Enable AP6XXX wifi module for rv1126/rv1109 evb board.

Signed-off-by: Weiwen Chen <cww@rock-chips.com>
Change-Id: I5293a8d4071716532a2d63f88e9c5c20ebd5fd36
---
 arch/arm/configs/rv1126_defconfig | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/rv1126_defconfig b/arch/arm/configs/rv1126_defconfig
index d63d43c4b94b..e2da202f9a14 100644
--- a/arch/arm/configs/rv1126_defconfig
+++ b/arch/arm/configs/rv1126_defconfig
@@ -147,7 +147,8 @@ CONFIG_STMMAC_ETH=y
 # CONFIG_WLAN_VENDOR_ZYDAS is not set
 # CONFIG_WLAN_VENDOR_QUANTENNA is not set
 CONFIG_WL_ROCKCHIP=y
-CONFIG_WIFI_LOAD_DRIVER_WHEN_KERNEL_BOOTUP=y
+CONFIG_WIFI_BUILD_MODULE=y
+CONFIG_AP6XXX=m
 # CONFIG_RTL_WIRELESS_SOLUTION is not set
 CONFIG_INPUT_EVDEV=y
 CONFIG_KEYBOARD_ADC=y
-- 
2.35.3

